Yoast/YoastSEO.js
# Add punctuation marks to addBoundary

## Summary

This PR can be summarized in the following changelog entry:

* Improves keyword recognition through adding Spanish inverted exclamation and question marks to the rules that determine word boundaries.

## Relevant technical choices: 

* I decided to not add `#%$` to word boundaries, because  those are in general not punctuation marks and can only mark word boundaries if used together with another word boundary (e.g., space, full stop). 

## Test instructions

This PR can be tested by following these steps:

* Write a text that uses `¿` and `¡` immediately before words, like _¿Cómo estás?_
* Make the word that follows one of these marks your keyword.
* Make sure that the word is correctly recognized in all relevant assessments.

Fixes #1003, Fixes #1609
